pages/c/_cluster/index.vue

```diff
@@ -132,15 +185,21 @@ export default {
     },
 
     async fetchClusterResources(type, opt = {}) {
-      try {
-        const resources = await this.$store.dispatch('cluster/findAll', { type, opt });
+      const schema = this.$store.getters['cluster/schemaFor'](type);
 
-        return resources;
-      } catch (err) {
-        console.error(`Failed fetching cluster resource ${ type } with error:`, err);
+      if (schema) {
+        try {
+          const resources = await this.$store.dispatch('cluster/findAll', { type, opt });
 
-        return [];
+          return resources;
+        } catch (err) {
+          console.error(`Failed fetching cluster resource ${ type } with error:`, err);
+
+          return [];
+        }
       }
+
+      return [];
     },
 
     async pollMetrics() {
```

Comment:
This needs to run only if the metrics schema exists and the rest of the page handle it not existing (e.g. single node rancher's "local" doesn't have it by default)

Reply:
👍